lib/muchos/ec2.py

```diff
@@ -322,8 +323,8 @@ def terminate(self):
             if path.isfile(self.config.hosts_path):
                 os.remove(self.config.hosts_path)
                 print("Removed hosts file at ", self.config.hosts_path)
-        else:
-            print("Aborted termination")
+            else:
```

Comment:
@plainolneesh you need to address this - I agree with @keith-turner that this change looks out of place and should be restored to what it was. If we are both missing a point, please clarify.

Reply:
I don't see line 326 in the current version. I think this was outdated. But if I am overlooking this, please let me know.

Reply:
I think it was fixed with the start/stop merge.

Reply:
Thank you @plainolneesh. I reviewed this thoroughly again, and I am convinced we need to back this out. To help things move along faster and to avoid any confusion, I have pushed [this commit](https://github.com/arvindshmicrosoft/fluo-muchos/commit/f0ee40078d672dfd438f92d2bc39d5da44494230) to my fork. Please take a look and merge if okay. After this, I am pretty much good to approve the PR. @keith-turner please do take a look, and LMK if you are okay with resolving this comment?